reference/shipping.v3.yml

```diff
@@ -3,13 +3,14 @@ info:
   version: ''
   title: Shipping V3
   description: V3 REST API shipping endpoints.
-  termsOfService: 'http://www.bigcommerce.com/terms' 
-  contact: 
-    name: BigCommerce 
-    url: 'https://www.bigcommerce.com' 
-    email: support@bigcommerce.com 
+  termsOfService: 'http://www.bigcommerce.com/terms'
+  contact:
+    name: BigCommerce
+    url: 'https://www.bigcommerce.com'
+    email: support@bigcommerce.com
 tags:
   - name: Customs Information
+  - name: Shipping Settings
```

Comment:
🍻 This seems like a strange tag - is this the usual approach? 

I could imagine `Shipping` being a relevant tag, less sure about `Shipping Settings`. Did you have guidance here @bc-andreadao?

Reply:
Thanks Brett, I originally had checkout settings there, but then @agchan12 suggested this is in the shipping domain with a single checkout setting in it for now. What are your thoughts @agchan12 ?

Reply:
I think @theromulans was more suggesting that the tag of `Shipping Settings` might be irrelevant and can be replaced to something more like `Shipping` instead.

For the tag recommendation though, they are used to group API endpoints logically, making the API documentation easier to navigate and understand. I am ok with keeping things generic and having the `shipping` tag separate. Keep in mind multiple tags can be assigned to an endpoint too.

Reply:
The list already includes `Customs Information` - this is why I'm asking for guidance from @bc-andreadao about what these values drive. Clearly our new Shipping Settings endpoint has no relevance to Customs Information either.

Reply:
This doc somewhat explains how tags work - https://idratherbewriting.com/learnapidoc/pubapis_openapi_step7_tags_object.html
in summary - the global tags group lists all the defined tags on the current doc page, each tag underneath a `path` section denotes that path section tag (which drives the nav menu and headings I believe, once the doc site is built), if that makes sense.

Reply:
Indeed, if this is driving the navigation and headings on our docs site then there's no question about the most appropriate tag being `Shipping Settings` for our two new operations.

For example, the current `Customs Information` outcome:

<img width="799" alt="Screenshot 2024-09-10 at 4 26 32 PM" src="https://github.com/user-attachments/assets/aa6b4ca2-f29c-4987-8778-9a85f57bd9fc">

Reply:
Yes, the tags create the dividers on the site (as shown in @theromulans screenshot). 

[Line 170](https://github.com/bigcommerce/docs/blob/d08d3a38e0eb2eda3eb08a5fd049568f3f2d871a/reference/shipping.v3.yml#L170) and [Line193](https://github.com/bigcommerce/docs/blob/d08d3a38e0eb2eda3eb08a5fd049568f3f2d871a/reference/shipping.v3.yml#L193) make it so that the `GET /shipping/settings` endpoint and `POST /shipping/settings` endpoint appear under the "Shipping Settings" divider. 

The global tags ([line 11](https://github.com/bigcommerce/docs/blob/d08d3a38e0eb2eda3eb08a5fd049568f3f2d871a/reference/shipping.v3.yml#L11)) is just a cumulative list of all the tags that we put in the doc. If we don't include it, we'll get a linting error.

I don't have an opinion about whether or not it should be "Shipping Settings" or "Settings"

Reply:
One minor detail is that the tag name also appears in the URL. 
![URL](https://github.com/user-attachments/assets/72dd88ca-250d-4f9e-9356-617f28f2aec0)

Reply:
Thanks @bc-andreadao, Shipping Settings looks good then, thanks for confirming how the tags are used.

Curiously, do you know if a tag's description is used (if we define one)?

Reply:
I actually am not sure about tag descriptions (only some parts of the OAS3 file spec are supported on our site, so it may or may not work)

Reply:
Thanks Andrea for the explanation and clarification on this. Staying with shipping settings here.